frontend/src/features/research/add/AddResearchForm.tsx

```diff
@@ -0,0 +1,107 @@
+import { mdiFolderText } from '@mdi/js';
+import Icon from '@mdi/react';
+import { InlineInput } from 'components/common/form/styles';
+import * as Styled from 'components/common/styles';
+import { Formik } from 'formik';
+import { Api_ResearchFile } from 'models/api/ResearchFile';
+import * as React from 'react';
+import { Button, Col, Row } from 'react-bootstrap';
+import { Prompt, useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { useAddResearch } from '../hooks/useAddResearch';
+import { AddResearchFileYupSchema } from './AddResearchFileYupSchema';
+import { ResearchForm } from './models';
+import ResearchProperties from './ResearchProperties';
+
+const AddResearchForm: React.FunctionComponent = () => {
+  const initialForm = new ResearchForm();
+  const history = useHistory();
+
+  const { addResearchFile } = useAddResearch();
```

Comment:
I'm guessing tests are coming with the add properties logic?

Reply:
Yes, at this point this does not add properties